src/commands/data/connectors/destroy.ts

```diff
@@ -11,23 +13,34 @@ export default class ConnectorsDestroy extends BaseCommand {
     },
   ]
 
+  static flags = {
+    confirm: flags.boolean({
+      description: 'confirms destroying the connector if passed in',
+      required: false,
+    }),
+  }
+
   static examples = [
     '$ heroku data:connectors:destroy gentle-connector-1234',
   ]
 
   async run() {
     const {args} = this.parse(ConnectorsDestroy)
     const connector = args.connector
+    const confirm = flags
```

Comment:
```suggestion
    const confirm = flags.confirm
```

I just tried this and it deleted the connector without asking. 😨 

```
$ myh help data:connectors:destroy
destroy a Data Connector

USAGE
  $ heroku data:connectors:destroy [CONNECTOR]

OPTIONS
  --confirm  confirms destroying the connector if passed in

EXAMPLE
  $ heroku data:connectors:destroy gentle-connector-1234


$ myh data:connectors:destroy powerful-connector-48451
Data Connector powerful-connector-48451 deleted successfully.
Destroying Data Connector... done
```